components/clarinet-deno/index.ts

```diff
@@ -166,28 +165,35 @@ export class Chain {
     return this.mineEmptyBlock(count);
   }
 
+  /**
+   * Call a read-only function
+   * @param contract Address of the contract implementing the function
+   * @param method The read-only function to call
+   * @param args Arguments to pass as clarity values
+   * @param sender Address of the caller
+   * @returns The result of th
+   */
```

Comment:
@mefrem 
Added JSDoc for this function.
It'll appear in the generated doc and could be added to more functions (`getAssetsMaps`, `callContract` etc)